src/components/SpaceBg/SpaceBg.stories.mdx

```diff
@@ -0,0 +1,22 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs'
+import { Box, Flex } from 'rebass'
+
+import { SpaceBg } from './SpaceBg'
+
+<Meta title="SpaceBg" component={SpaceBg} />
+
+# SpaceBg
+
+SpaceBg is the animated background with planets on swapr.
+
+<Canvas>
+  <Story name="SpaceBg">
+    <Box bg="#0c0b11" width="100vw" height="100vh">
+      <SpaceBg>
+        <Flex width="100%" alignItems="center">
+          Hey
+        </Flex>
+      </SpaceBg>
+    </Box>
+  </Story>
+</Canvas>
```

Comment:
Hmm what was the reasoning behind having this background as storybook component? Also would be better to make separate folder for it...

Reply:
yeah, maybe we don't need it here. I'll remove it.